src/main/java/nl/igorski/example/MWEngineActivity.java

```diff
@@ -64,6 +66,23 @@ public void onCreate( Bundle savedInstanceState )
         super.onCreate( savedInstanceState );
         setContentView( R.layout.main );
 
+        // these may not necessarily all be necessary for your use case (e.g. if you're not
+        // recording from device audio inputs or reading/writing files) but are here for documentation purposes
+
+        if ( android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.M ) {
+            String[] PERMISSIONS = {
+                    Manifest.permission.RECORD_AUDIO,
+                    Manifest.permission.READ_EXTERNAL_STORAGE,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE
+            };
+
```

Comment:
Put in the example Activity to prevent frustration when trying to implement recording and storage functions ;)